src/applications/vaos/services/appointment/transformers.js

```diff
@@ -396,17 +394,31 @@ function setContained(appt) {
     }
     case APPOINTMENT_TYPES.ccRequest: {
       return appt.ccAppointmentRequest.preferredProviders.map(provider => {
-        return {
+        let obj = {
           actor: {
             name: provider.practiceName,
-            address: {
-              line: provider.address?.street,
-              city: provider.address?.city,
-              state: provider.address?.state,
-              postalCode: provider.address?.zipCode,
-            },
+            // TODO: Map to participant.actor.Practitioner field.
+            firstName: provider.firstName,
+            lastName: provider.lastName,
           },
         };
+
+        if (provider.address) {
+          const address = {
+            line: [provider.address?.street],
+            city: provider.address?.city,
+            state: provider.address?.state,
+            postalCode: provider.address?.zipCode,
+          };
+          obj = {
```

Comment:
Since you created `obj`, you could do `obj.actor.address = address;`

Reply:
Done